examples/config.hocon.kinesis.minimal

```diff
@@ -0,0 +1,25 @@
+collector {
+  interface = "0.0.0.0"
+  port = 80
+
+  streams {
+    good = "{{good}}"
+    bad = "{{bad}}"
+
+    sink {
+      enabled = kinesis
+      region = "{{region}}"
```

Comment:
I never liked this `{{ }}` syntax.  It doesn't mean anything, and we don't use it any other example file.

The purpose of these files is to be a demonstrative example of typical values.  I think it is OK to put a real example, e.g. `eu-central-1`.  See [here as an example](https://github.com/snowplow-incubator/snowplow-postgres-loader/blob/0.3.1/config/config.kinesis.minimal.hocon#L6).

Same applies for all other keys where you have `{{ }}`.